v2.0/enable-node-map.md

```diff
@@ -214,3 +215,35 @@ The **Node Map** is displayed only for the locality levels that have latitude/lo
 |  Asia Pacific (Sydney) | `INSERT into system.locations VALUES ('region', 'ap-southeast-2', -33.86882, 151.209296)` |
 |  Asia Pacific (Mumbai) | `INSERT into system.locations VALUES ('region', 'ap-south-1', 19.075984, 72.877656)` |
 |  South America (São Paulo) | `INSERT into system.locations VALUES ('region', 'sa-east-1', -23.55052, -46.633309)` |
+
+### Azure locations
```

Comment:
@a-robinson, this is awesome, but to make it easy to update this, can you please:

1. Make this azure table an include.
2. Add the python script to the `scripts/` directory. 

It'd be even better for the script to edit the relevant include instead of printing the table, but that's just a nice to have. Eventually, we could just wrap this into our TC builds.

Reply:
@a-robinson, if you would prefer that I create the script separately, this PR LGTM.

Reply:
Done, in the sense that it's all runnable as a single command. I'm not sure whether more would be needed to fit it into a TC build.

Reply:
Thank you!